### PR TITLE
compute: use map for builtin membership check

### PR DESCRIPTION
**Pull request by rvantonder**
As in title, addresses some feedback I fix up while merging stack.

### CODE REVIEW

File: internal/compute/template.go [hunk 1]
```diff
@@ -191,23 +190,16 @@ type MetaEnvironment struct {
...
-	return false
+var builtinVariables = map[string]struct{}{
+	"repo":    empty,
+	"path":    empty,
+	"content": empty,
+	"commit":  empty,
+	"author":  empty,
+	"date":    empty,
+	"email":   empty,
 }
```
Comment (camdencheek) [inline internal/compute/template.go:203]: Oh, I was thkinking the values would be the template variable

```suggestion
var builtinVariables = map[string]struct{}{
	"repo":    `{{ .Repo }}`,
	"path":   `{{ .Path }}`,
	"content": `{{ .Content }}`,
	"commit":  `{{ .Commit }}`,
	"author":  `{{ .Author }}`,
	"date":   `{{ .Date }}`,
	"email":   `{{ .Email }}`,
}
```

Comment (rvantonder) [reply internal/compute/template.go:203]: Ah! yeah we can do this way :-) This nudges me even more to just yoink some substitution code and write my own :P

Comment (camdencheek) [reply internal/compute/template.go:203]: Cool. As always, feel free to ignore if you're not at the point this makes sense.

**Review by camdencheek — APPROVED** [review]

**Comment by camdencheek** [comment]
Whoops, thought I reviewed before

**Pull request merged**